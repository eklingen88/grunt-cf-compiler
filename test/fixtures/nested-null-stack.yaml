AWSTemplateFormatVersion: 2010-09-09
Description: CloudFormation exports
Conditions:
  HasNot: !Equals
    - 'true'
    - 'false'
Resources:
  NullResourceMain:
    Type: 'Custom::NullResource'
    Condition: HasNot
  NestedNullStack:
    Type: 'AWS::CloudFormation::Stack'
    Properties:
      TemplateURL: './null-stack.yaml'
Outputs:
  ExportsStackName:
    Value: !Ref 'AWS::StackName'
    Export:
      Name: !Sub 'ExportsStackName-${AWS::StackName}'
