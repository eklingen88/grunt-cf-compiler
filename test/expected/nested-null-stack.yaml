AWSTemplateFormatVersion: 2010-09-09
Description: CloudFormation exports
Conditions:
  HasNot:
    Fn::Equals:
      - "true"
      - "false"
Resources:
  NullResourceMain:
    Type: Custom::NullResource
    Condition: HasNot
  NullResource:
    Type: Custom::NullResource
    Condition: HasNot
Outputs:
  ExportsStackName:
    Value:
      Ref: AWS::StackName
    Export:
      Name:
        Fn::Sub: ExportsStackName-${AWS::StackName}
